.github/workflows/build.yml

```diff
@@ -19,6 +19,70 @@ jobs:
 
     name: ${{ matrix.name }}
 
+    env:
+      ARTIFACTS_DIR: exported-artifacts
+
+    container:
+      image: quay.io/centos/centos:${{ matrix.container-name }}
```

Comment:
Please try to use oVirt build container:

https://github.com/oVirt/ovirt-engine-extensions-api/blob/master/.github/workflows/build.yml#L18
https://github.com/oVirt/ovirt-engine-extensions-api/blob/master/.github/workflows/build.yml#L29

 It should have installed all required dependencies as soon as you have RPM published to COPR:

https://github.com/oVirt/buildcontainer/blob/main/Dockerfile.stream8#L29